cmd/argoexec/commands/emissary_windows_test.go

```diff
@@ -17,27 +17,27 @@ func TestEmissary(t *testing.T) {
 	includeScriptOutput = true
 
 	err := os.WriteFile(varRunArgo+"/template", []byte(`{}`), 0o600)
-	assert.NoError(t, err)
+	require.NoError(t, err)
```

Comment:
this file is failing lint as it's missing an import for this

Reply:
Ugh. Windows.

Yeah, I'd been working off the basis that my compiler would find these.

Reply:
ah yeah, this has a build tag so locally you might miss that if you're not doing a Windows lint/build